docs/integrations/sources/mssql.md

```diff
@@ -1,4 +1,9 @@
 # Microsoft SQL Server (MSSQL)
+Airbyte's certified MySQL connector offers the following features:
```

Comment:
Airbyte's certified MSSQL connector offers the following features:

Reply:
😳